easystacks/software.eessi.io/2023.06/grace/eessi-2023.06-eb-4.9.4-2023a.yml

```diff
@@ -17,3 +17,20 @@ easyconfigs:
   - LHAPDF-6.5.4-GCC-12.3.0.eb
   - LoopTools-2.15-GCC-12.3.0.eb
   - R-4.3.2-gfbf-2023a.eb
+  - Boost-1.82.0-GCC-12.3.0.eb:
+      options:
+        # source URLs for Boost have changed, corresponding PR is
+        # https://github.com/easybuilders/easybuild-easyconfigs/pull/22157
+        # Boost is a dependency of AOFlagger
+        from-commit: 5bebccf792ccf35a8ee3250bc8fed86dff5d5df9
+  - Boost.Python-1.82.0-GCC-12.3.0.eb:
+      options:
+        # source URLs for Boost.* have changed, corresponding PR is
+        # https://github.com/easybuilders/easybuild-easyconfigs/pull/22240
+        # Boost.Python is a dependency of AOFlagger
```

Comment:
Little comment, in this case Boost.Python is likely a dependency of another package.